### PR TITLE
ConsoleLog → ConsoleLogger

### PR DESCRIPTION
**Pull request by tayloraswift**
update ConsoleLog → ConsoleLogger

### CODE REVIEW

File: Sources/ConsoleKitTerminal/Docs.docc/index.md [hunk 1]
```diff
@@ -8,4 +8,4 @@
 
 * Utilities for sending text (including styles and colors, when supported) to a terminal.
 * Utilities for reading input from a terminal.
-* ``ConsoleLog`` and ``ConsoleFragmentLogger``, [SwiftLog](https://github.com/apple/swift-log) `LogHandler` implementations for customizable logging to a console.
+* ``ConsoleLogger`` and ``ConsoleFragmentLogger``, [SwiftLog](https://github.com/apple/swift-log) ``LogHandler`` implementations for customizable logging to a console.
```
Comment (gwynne) [inline Sources/ConsoleKitTerminal/Docs.docc/index.md:11]:
```suggestion
* ``ConsoleLogger`` and ``ConsoleFragmentLogger``, [SwiftLog](https://github.com/apple/swift-log) `LogHandler` implementations for customizable logging to a console.
```
The lack of a symbol link on `LogHandler` is deliberate - DocC still doesn't support cross-module linking, so the link won't work anyway.

**Review by gwynne — CHANGES_REQUESTED** [review]

File: Sources/ConsoleKitTerminal/Docs.docc/index.md (hunk 1 above)
Comment (tayloraswift) [reply Sources/ConsoleKitTerminal/Docs.docc/index.md:11]: is this causing DocC errors? other rendering engines (like Swiftinit) have no problem resolving cross-package links

Comment (gwynne) [reply Sources/ConsoleKitTerminal/Docs.docc/index.md:11]: TBH I don't recall offhand whether it causes an error or just a warning.

**Review by gwynne — APPROVED** [review]
Docs-only change, no release needed

**Pull request merged**